docs/runtime/workers.md

```diff
@@ -233,8 +236,8 @@ the `deno.permissions` option in the worker API.
   });
   ```
 
-- You can disable the permissions of the worker all together by passing false to
-  the `deno.permissions` option.
+- You can disable the permissions of the worker all together by passing `"none"`
+  to the `deno.permissions` option.
```

Comment:
I don't think this is right - shouldn't this be `false`? CC @crowlKats @Soremwar

Reply:
@bartlomieju it seems that individual permissions take a boolean whereas the actual `permissions` property takes a "none" string, alongside a "inherit" string. ideally it would be a boolean, but what should `true` then be? allow all perms? seems kinda weird

Reply:
Ooops, you're right, thanks for clarifying crowlKats